src/Open3D/Visualization/Rendering/RendererStructs.h

```diff
@@ -26,11 +26,12 @@
 
 #pragma once
 
-#include "RendererHandle.h"
-
 #include <json/json.h>
+
```

Comment:
delete empty line?

Reply:
inserted by clang-format. (see above)

Reply:
this turns out to be a difference between the way clang-format-5 and clang-format-9 implement SortIncludes. Empty line removed.